aws-serverless-java-container-core/src/main/java/com/amazonaws/serverless/proxy/internal/servlet/AwsHttpServletResponse.java

```diff
@@ -213,7 +213,11 @@ public void addDateHeader(String s, long l) {
     @Override
     public void setHeader(String s, String s1) {
         if (!canSetHeader()) return;
-        setHeader(s, s1, true);
+        if ("Content-Type".equalsIgnoreCase(s)) {
```

Comment:
Can you please align this check with the one below in `addHeader`?

Reply:
Done